src/renderer/data/accounts/getAccount.ts

```diff
@@ -16,20 +16,32 @@ const HEXDUMP_BYTES = 512;
 //       Also use that to decide if we need to do a validator is available check or not - if we're watching changes, then we already know...
 
 const cache = new LRUCache<string, AccountInfo>({
-  maxSize: 100,
-  entryExpirationTimeInMS: 1000,
+  maxSize: 500,
+  entryExpirationTimeInMS: 60000,
 });
 
+export const getAllAccounts = (): AccountInfo[] => {
+  const list: AccountInfo[] = [];
```

Comment:
yeah, this is horrid, but seems to work about 70% of the time - the other 30%, everything hangs, so I'm guessing the "for sorting" list should be created on the fly, and then it's just an ordered array lookup